### ISSUE TITLE
nn.CrossEntropyLoss的标签类型问题导致Loss计算错误

### ISSUE THREAD

**Issue by XiangXiangHotSpot** [RenqiChen/Genomic-Selection#5]
在dataset_rise3k_45.py文件中，label_map函数返回的是原始标签的float64格式。然而，nn.CrossEntropyLoss函数期望的输入是类别索引，而非连续值标签。因此，当前的实现会导致类型不匹配，从而引发错误。是否应该将label_map函数的返回值改为类别的整数索引，以适配nn.CrossEntropyLoss的要求。

可以将
```
        label_map = {}
        for index, row in sample_list.iterrows():
            if row['NAME'] in name2id.keys() and pd.notna(row[task_name]):
                sample_id = name2id[row['NAME']]
                label = row[task_name]
                label_map[sample_id] = label

        train_d, test_d = apply_KFold(data_root, label_map, fold_root, which_k)
```
修改成
```
        label_map = {}
        for index, row in sample_list.iterrows():
            if row['NAME'] in name2id.keys() and pd.notna(row[task_name]):
                sample_id = name2id[row['NAME']]
                label = row[task_name]
                label_map[sample_id] = label

        # 提取标签值
        labels = list(label_map.values())
        # 获取唯一标签并排序
        unique_labels = sorted(set(labels))
        # 创建映射字典
        label_dic = {label: idx for idx, label in enumerate(unique_labels)}
        # 创建映射后的字典
        mapped_data = {key: label_dic[value] for key, value in label_map.items()}

        train_d, test_d = apply_KFold(data_root, mapped_data, fold_root, which_k)
```

**Comment by RenqiChen** [comment]
We did not encounter this issue when using the labels from the rice3k dataset (ce_loss); however, we found that it was necessary to additionally include a `float()` conversion when using the labels from the wheat3k dataset (mse_loss).
Anyway, if you can run our program successfully in this manner, I believe it's acceptable.
